.travis.yml

```diff
@@ -1,19 +1,31 @@
 language: php
 
 php:
-  - 5.3
```

Comment:
If we just completely ignore two versions of php, shouldn't it be a major bump? Technically it's not BC